db/schema/schema_elf.sql

```diff
@@ -0,0 +1,79 @@
+DROP TABLE IF EXIST laptops CASCADE;
+DROP TABLE IF EXIST ads CASCADE;
+DROP TABLE IF EXIST laptop_images CASCADE;
+DROP TABLE IF EXIST users CASCADE;
+DROP TABLE IF EXIST user_favorites CASCADE;
+DROP TABLE IF EXIST messages CASCADE;
+DROP TABLE IF EXIST chats CASCADE;
+DROP TABLE IF EXIST chat_members CASCADE;
+
+CREATE TABLE laptops (
+  id SERIAL PRIMARY KEY NOT NULL,
+  brand_name VARCHAR(225) NOT NULL,
+  screen_size INTEGER NOT NULL DEFAULT 0,
+  condition VARCHAR(225) NOT NULL,
+  memory INTEGER NOT NULL DEFAULT 0
+  price INTEGER NOT NULL DEFAULT 0,
+  storage_size INTEGER NOT NULL DEFAULT 0,
+  storage_type VARCHAR(225) NOT NULL
+);
+
+CREATE TABLE ads (
+  id SERIAL PRIMARY KEY NOT NULL,
+  laptop_id INTEGER REFERENCES laptops(id) ON DELETE CASCADE,
+  user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+  title VARCHAR(225) NOT NULL,
+  description TEXT,
+  is_featured BOOLEAN NOT NULL DEFAULT FALSE,
+  status VARCHAR(225),
+  created_at DATE NOT NULL,
+  updated_at DATE NOT NULL,
+  laptop_image_id INTEGER REFERENCES laptop_images(id) ON DELETE CASCADE,
+);
+
+CREATE TABLE laptop_images (
+  id SERIAL PRIMARY KEY NOT NULL,
+  laptop_id INTEGER REFERENCES laptops(id) ON DELETE CASCADE,
+  url VARCHAR(225) NOT NULL,
+  created_at DATE NOT NULL,
+  updated_at DATE NOT NULL
+);
+
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(225) NOT NULL,
+  email VARCHAR(225) NOT NULL,
+  password VARCHAR (225) NOT NULL,
+  contact_phone VARCHAR(15) NOT NULL,
+  phopo_url VARCHAR(2000)

```

Comment:
this is photo_url
and please add one column for is_admin which is Boolean. default is false; THank you @Valera-Neg .. Awesome work,